tools/generate_install_yamls_playbooks.py

```diff
@@ -0,0 +1,33 @@
+# Python Script to generate playbooks
+import os
+import sys
+
+script_path = os.path.realpath(os.path.dirname(__file__))
+install_yamls_tasks_dir = os.path.join('ci_framework', 'roles', 'use_install_yamls', 'tasks')
+playbooks_dir = os.path.join(os.path.dirname(script_path), install_yamls_tasks_dir)
```

Comment:
ah, that won't work with the new tree I think?

Also, we may want to push the generated content somewhere else, like `{{ cifmw_basedir }}/artifacts/make_installyamls/tasks` - and add that path to the ansible env... ?

Reply:
Done